ktor-server/ktor-server-plugins/ktor-server-single-page/jvm/src/io/ktor/plugins/SinglePageApplication.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.plugins
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.http.content.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.util.*
+import io.ktor.util.pipeline.*
+import java.io.*
+
+/**
+ * This plugin provides a single page application
```

Comment:
This plugin -> A plugin (according to 'API Docs guidelines)
single page -> single-page

So, I suggest smth like this:
> A plugin that allows you to serve a single-page application

It would be also useful to have a simple code snippet in a plugin description. The most simple config is enough.